marked-element.html

```diff
@@ -305,10 +314,13 @@
         // Note: if we are using the file:// protocol, the status code will be 0
         // for all outcomes (successful or otherwise).
         if (status === 0 || (status >= 200 && status < 300)) {
+          this.sanitize = true;
           this.markdown = e.target.response;
         } else {
           this._handleError(e);
         }
+
+        this.fire('marked-loadend', e);
```

Comment:
necessary because data binding does not have a `this.xhr` so you can't attach a `loadend` listener.